conductor_api/src/config.rs

```diff
@@ -478,6 +478,15 @@ impl Configuration {
         self.dnas.iter().find(|dc| &dc.id == id).cloned()
     }
 
+    /// Returns the DNA configuration with the given ID if present
```

Comment:
I think this comment got copy-pasted unchanged :)
```suggestion
    /// Updates the hash of the DNA with the given ID, returns true if it was present.
```